lte/gateway/c/oai/tasks/nas/esm/esm_data_context.c

```diff
@@ -114,6 +115,10 @@ void free_esm_context_content(esm_context_t *esm_ctx)
 {
   if (esm_ctx) {
     nas_stop_T3489(esm_ctx);
+    if (esm_ctx->esm_proc_data) {
+      bdestroy_wrapper(&esm_ctx->esm_proc_data->apn);
```

Comment:
Do we need to destroy pdn_addr here too?

Reply:
Currently we are not allocating memory for pdn_addr within esm_proc_data.
It will be used, if UE sends it's IP address to MME. Since memory is not allocated I didn't add code to destroy pdn_addr